registry/config.go

```diff
@@ -143,6 +143,13 @@ func (config *serviceConfig) LoadInsecureRegistries(registries []string) error {
 
 skip:
 	for _, r := range registries {
+		if strings.Contains(r, "://") {
+			// Insecure registry does not contain a scheme.
+			// before returning err, roll back to original data
+			config.ServiceConfig.InsecureRegistryCIDRs = originalCIDRs
+			config.ServiceConfig.IndexConfigs = originalIndexInfos
+			return fmt.Errorf("insecure registry %s should not contain a scheme '%s://'", r, strings.Split(r, "://")[0])
```

Comment:
Oh I didn't see the *warning* part of the comment. Let me update the PR shortly.